lib/datasource.js

```diff
@@ -348,6 +348,14 @@ DataSource.prototype.setup = function(name, settings) {
     if (settings.initialize) {
       connector = settings;
     } else if (settings.connector) {
+      // some connectors are aliases to loopback-component-storage
+      if (['object-storage'].indexOf(settings.connector) > -1) {
```

Comment:
It definitely make sense to keep juggler as loosely coupled from the LoopBack ways, as possible.

The other option would be to implement this in `loopback`. Will it be acceptable there? Aliases have to be handled at some level, though.